docs/plugins/composable_resources.md

```diff
@@ -197,6 +197,13 @@ To modify an existing passthru resource to be written to only after other eligib
 irods@hostname:~/ $ iadmin modresc newResc context 'write=0.4;read=1.0'
 ```
 
+Setting a `read` or `write` weight to `0` will prevent that action from occurring on the branch of the resource tree under the passthru (because a `0` vote signals that it cannot service the incoming request).  The following will 'turn off' the resource `newResc` (a policy-based way of marking a resource "down"):
```

Comment:
Should this say 0.0 instead of 0? I'm guessing 0 also works, but the example below says 0.0.

Also, would it be worth mentioning that this "turns off" the entire subtree under `newResc` (not just `newResc`)?

Reply:
Consistency is good. I vote for 0.0 since the example uses that.

Reply:
yeah, went back and forth over it - but looking at it again... seems like a typo rather than intentional.

will use 0.0... a bit more refined :)

will mention "(and subtree)"

Reply:
tried zero rather than 0.0 in the running text.